addons/hr/hr.py

```diff
@@ -316,10 +316,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user_obj = self.pool.get('res.users').browse(cr, uid, user_id, context=context)
```

Comment:
Yes please, cherry-pick it for me if you can.
I'm new to Odoo, need to figure out what is what's and where have to be committed, but I will do my best to help you out in the future. I will remember all advises for the next commit.